WordPress/src/main/java/org/wordpress/android/ui/mysite/ScanAndBackupSource.kt

```diff
@@ -22,7 +22,8 @@ class ScanAndBackupSource
             val itemsVisibility = jetpackCapabilitiesUseCase.getJetpackPurchasedProducts(site.siteId)
             emit(
                     JetpackCapabilities(
-                            scanAvailable = scanScreenFeatureConfig.isEnabled() && itemsVisibility.scan,
+                            scanAvailable = scanScreenFeatureConfig.isEnabled() && itemsVisibility.scan &&
```

Comment:
1) Minor (🔍): Since the `&&` logic is getting more complicated, I would suggest splitting it in multiple line to increase readability. For example:
```
scanAvailable = scanScreenFeatureConfig.isEnabled() &&
        itemsVisibility.scan &&
        !site.isWPCom &&
        !site.isWPComAtomic,
backupAvailable = backupScreenFeatureConfig.isEnabled() &&
        itemsVisibility.backup
```
2) Minor (🔍) It seems that this class can be reformatted.
3) Minor (🔍) The `@Inject constructor` line can be merged with its above line. This is not your change so feel free to ignore.

PS: Readability is very subjective so please ignore this comment if you feel about this differently.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13962/commits/fdc9032df95dba6292afcbf1b076450c0809e79c

Reply:
👋 @malinajirka !

Thanks for this. 🙏 Just a quick note that the `&&` must be at the end of the line. A new line must not begin with `&&`. At least this is how I know this convention to be. Is this not the case here?

Reply:
I'm never sure which one is correct and I always pick the wrong one :D.

Reply:
I have `Detekt` for that! 😄 

PS: Soon you will too! 🎉

Reply:
Fixed in 078f00e2583508fd0b93227b89e719c5863eedba